### PR TITLE
Fix unknown CMake command "CPMFindPackage"

### PR DESCRIPTION
**Pull request by standbyme**
Build RMM error:
```shell
cmake .. -DCMAKE_INSTALL_PREFIX=/install/path
```

```
-- Downloading CPM.cmake to ...
CMake Error at cmake/Modules/RMM_thirdparty.cmake:5 (CPMFindPackage):
  Unknown CMake command "CPMFindPackage".
Call Stack (most recent call first):
  CMakeLists.txt:22 (include)
```

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: cmake/Modules/CPM.cmake [hunk 1]
```diff
@@ -1,6 +1,8 @@
-set(CPM_DOWNLOAD_VERSION c3c7e2d9a3b890511891ffc415d7ea23c689068a) # 0.27.3
+set(CPM_DOWNLOAD_VERSION 0.28.4)
```
Comment (kkraus14) [inline cmake/Modules/CPM.cmake:1]: We use a git commit hash and the raw link to make sure that the upstream tag doesn't change out from underneath us. Any chance you could update this to follow that pattern?

**Review by kkraus14 — CHANGES_REQUESTED** [review]

File: cmake/Modules/CPM.cmake (hunk 1 above)
Comment (standbyme) [reply cmake/Modules/CPM.cmake]: Finished.
Thanks for your comment : )

**Comment by harrism** [comment]
@germasch may want to review.

**Review by germasch — APPROVED** [review]
Looks good to me ;)

**Comment by harrism** [comment]
ok to test

**Comment by standbyme** [comment]
@harrism hello, is there other problems? : )

**Review by kkraus14 — APPROVED** [review]

**Comment by kkraus14** [comment]
@gpucibot merge

**Pull request merged**